src/main/scala/rules/SymbolicExecutionRules.scala

```diff
@@ -21,21 +21,28 @@ trait SymbolicExecutionRules {
         wrapped
       case _ => ve
     }
+    //for some reason we cannot pass --coutereaxmples as an argument...

```

Comment:
Typo? `coutereaxmples`

Reply:
The issue arose from using a single string e.g. "--counterexample mapped" instead of "--counterexample" , "mapped"  as argument.
(deleted comment)